destination.go

```diff
@@ -298,7 +298,7 @@ func (d *Destination) formatUpsertQuery(
 	colArgs, valArgs := d.formatColumnsAndValues(key, payload)
 
 	return d.stmtBuilder.
-		Insert(tableName).
+		Insert(`"` + tableName + `"`).
```

Comment:
@derElektrobesen thanks for the contribution! I have two notes here:
1. let's have a separate method to add the quotes, we have many uses for it. [not picky about this one, but could help with handling special cases, if any]
2. we need to ensure this works for any name, what if the table name has a quote `"` in the middle, or a `'` ? plz double-check for special characters and add some tests for those cases.

Reply:
Fixed
Check again please